tests/models/whisper/test_modeling_whisper.py

```diff
@@ -3231,3 +3231,15 @@ def test_retain_grad_hidden_states_attentions(self):
     @unittest.skip("The model doesn't support fast init from base")
     def test_save_load_fast_init_from_base(self):
         pass
+
+    @unittest.skip("Duplicated test with WhisperModelTester")
```

Comment:
> WhisperModelTester
I guess you mean `WhisperModelTest` ?

Reply:
BTW, `all_generative_model_classes` are different in these 2 test classes? Are they really duplicated?

Reply:
Yeah there is another issue which is the FA2 common tests not being adapted for these classes, I clarified the comment a bit

Reply:
updated.